packages/zoe/src/contracts/mintAndSellNFT.js

```diff
@@ -23,16 +23,14 @@ import '../../exported';
  * allows selling the tickets that were produced. You can reuse the ticket maker
  * to mint more tickets (e.g. for a separate show.)
  *
- * @param {ContractFacet} zcf
+ * @type {ContractStartFn}
  */
-const makeContract = zcf => {
-  const { terms } = zcf.getInstanceRecord();
-  const { tokenName = 'token' } = terms;
-
+const start = zcf => {
+  const { tokenName = 'token' } = zcf.getTerms();
   // Create the internal token mint
   const { issuer, mint, amountMath: tokenAmountMath } = makeIssuerKit(
```

Comment:
Should this have been turned into a synchronous mint?

Reply:
No, it is being sent to another contract, and there's no such thing as synchronous mint between contracts